newdle/client/src/components/AvailabilityRing.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+AvailabilityRing.defaultProps = {
```

Comment:
I'd move this after the function (while it works thanks to hoisting, it's a bit weird to access something before its definition)